server/index.js

```diff
@@ -11,15 +11,16 @@ app.use(`/restaurants/:id/`, express.static(`${__dirname}/../client/dist`));
 app.use(bodyParser());
 app.use(cors());
 
-app.get(`/api/reservations/restaurantID=:restaurantID&date=:date`, (req, res) => {
+app.get('/api/reservations/restaurantID=:restaurantID&date=:date', (req, res) => {
```

Comment:
Just wondering, how did you decide to use query strings on line 14 instead of regular parameters?